scholia/app/templates/author_curation.html

```diff
@@ -55,11 +49,16 @@ <h2 id="missing-citing-author-items-header">Missing citing author items</h2>
 
 <table class="table table-hover" id="missing-citing-author-items-table"></table>
 
+{% endblock %}
 
-<h2 id="missing-topic-header">Authored works with missing topics</h2>
 
-<table class="table table-hover" id="missing-topic-table"></table>
+{% block wikidata %}
 
+<p>For example, here are authored works by {{q}} that do not have topics. You can improve the data by clicking the link
+  on the right, scrolling to "+ add statement" and adding a "main subject" if you know it:</p>
```

Comment:
The ```main subject``` part should probably link to [P921](https://www.wikidata.org/wiki/Property:P921) to assist with cases when people are editing Wikidata in a non-English language, where ```P921``` would still work but ```main subject``` would not.